lib/licenses.nix

```diff
@@ -936,6 +936,11 @@ lib.mapAttrs mkLicense ({
     fullName = "Microsoft Public License";
   };
 
+  mvt = {
+    fullName = "MVT License 1.1";
+    url = "https://docs.mvt.re/en/latest/license/";
+  };
```

Comment:
There's no need to add a licenses.nix entry for a bespoke, non-free license.  Just set it to `licenses.unfree`.